### ISSUE TITLE
Deleting a system from web.fluffi buggy

### ISSUE THREAD

**Issue by joyride9999** [siemens/fluffi#142]
Trying to delete a system from web.fluffi ends up with error. Sometime happened that i could not see it on web.fluffi... but the system was in the database and could not add it again with the same name.
```
172.18.0.1 - - [16/Apr/2020:11:53:32 +0300] "GET / HTTP/1.0" 200 9066 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:75.0) Gecko/20100101 Firefox/75.0" "-"
[pid: 119|app: 0|req: 19/33] 172.18.0.1 () {42 vars in 738 bytes} [Thu Apr 16 11:53:32 2020] GET / => generated 9066 bytes in 33 msecs (HTTP/1.0 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 130|app: 0|req: 7/34] 172.18.0.1 () {44 vars in 786 bytes} [Thu Apr 16 11:53:47 2020] GET /systems => generated 22078 bytes in 933 msecs (HTTP/1.0 200) 3 headers in 96 bytes (1 switches on core 0)
172.18.0.1 - - [16/Apr/2020:11:53:48 +0300] "GET /systems HTTP/1.0" 200 22078 "http://web.fluffi/" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:75.0) Gecko/20100101 Firefox/75.0" "-"
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./app/views.py", line 1247, in removeSystemFromPolemarch
    removeResult = ANSIBLE_REST_CONNECTOR.removeSystem(hostName)
  File "./app/utils/ansible.py", line 156, in removeSystem
2020/04/16 11:53:58 [error] 12#12: *127 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: , request: "GET /systems/removeSystem/conan HTTP/1.0", upstream: "uwsgi://unix:///tmp/uwsgi.sock:", host: "localhost:8880", referrer: "http://web.fluffi/systems"
    for system in systems:
TypeError: 'NoneType' object is not iterable
[pid: 130|app: 0|req: 8/35] 172.18.0.1 () {44 vars in 831 bytes} [Thu Apr 16 11:53:53 2020] GET /systems/removeSystem/conan => generated 0 bytes in 5025 msecs (HTTP/1.0 500) 0 headers in 0 bytes (0 switches on core 0)
172.18.0.1 - - [16/Apr/2020:11:53:58 +0300] "GET /systems/removeSystem/conan HTTP/1.0" 502 173 "http://web.fluffi/systems" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:75.0) Gecko/20100101 Firefox/75.0" "-"
```

**Comment by joyride9999** [comment]
![VirtualBox_ubuntu 18 04 main_16_04_2020_12_07_33](https://user-images.githubusercontent.com/26308356/79437563-04019780-7fdb-11ea-8e3b-a43aae777e03.png)

**Comment by jones718** [comment]
This will be fixed by pull request #147.
